VARIABLES.md

```diff
@@ -1,98 +1,107 @@
 # Files and Variables
 
-A list of files within the project which the contain variables that need to be changed.
+A list of files within the project which the contain variables that need to be changed. The Encryption column denotes if the variable will be encrypted (tick), or stored in plaintext (blank).
+
+## Vault Password
+
+Ansible Vault password used to encrypt all of the following variables. This password should be stored securely for future use as it does not after project execution.
+
+| Variable | Description | Encrypted |
+| ------ | ----------- | ----------- |
+| `vault_password`: | The vault password used for variable encryption |  |
 
 ## aws_credentials.yml
 
 Credentials of the AWS account to be used to provision a new cluster. The credentials for multiple AWS environments can be stored in this file, with each environment required to be added to the `aws_credential_list`. The target AWS environment is specified when running the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) projects playbooks. An example is provided within the file.
 
-| Variable | Description |
-| ------ | ----------- |
-| `aws_credential_list: -"<CHANGEME>"`: | AWS credential name list (`dev`, `test` etc.) |
-| `<CHANGEME>_access_key_id: <CHANGEME>`: | AWS access key ID |
-| `<CHANGEME>_secret_access_key_id: <CHANGEME>`: | AWS secret key ID |
+| Variable | Description | Encrypted |
+| ------ | ----------- | ----------- |
+| `aws_credential_list: - "<CHANGEME>"`: | AWS credential name list (`dev`, `test` etc.) |  |
+| `access_key_id: <CHANGEME>`: | AWS access key ID | ✔ |
+| `secret_access_key_id: <CHANGEME>`: | AWS secret key ID | ✔ |
 
 ## cluster.yml
 
 Configuration variables required by the `provisioning_vars.yml.j2` file to provision of an Openshift cluster.
 
-| Variable | Description |
-| ------ | ----------- |
-| `cluster_pod_openshift_user`:   | Openshift cluster pod username |
-| `cluster_pod_openshift_pass`:   | Openshift cluster pod password |
-| `cluster_credential_bundle_aws_name`: | AWS cluster credential bundle name |
+| Variable | Description | Encrypted |
+| ------ | ----------- | ----------- |
+| `cluster_credential_git_repo`:  | Github project URL |  |
```

Comment:
Might be worth elaborating on this a bit more, the Github project URL of what? Itself?